tensorflow/contrib/gdr/gdr_memory_manager.cc

```diff
@@ -276,8 +275,8 @@ Status GdrMemoryManager::Init() {
     };
     GPUProcessState::singleton()->AddGPUAllocVisitor(bus_id,
                                                      cuda_alloc_visitor);
-    GPUProcessState::singleton()->AddCUDAHostAllocVisitor(bus_id,
-                                                          alloc_visitor);
+    GPUProcessState::singleton()->AddCUDAHostAllocVisitor(0, alloc_visitor);
+    GPUProcessState::singleton()->AddCUDAHostAllocVisitor(1, alloc_visitor);
```

Comment:
This seems fine for now.  NUMA specific allocation has not yet been turned on, pending bringing TF's eigen version up to date.   It might be to add a visitor for every available NUMA node by using port::NUMANumNodes.

Reply:
thanks for the tip. added it to latest commit.